zebra/zserv.c

```diff
@@ -945,6 +945,43 @@ static int zsend_ipv4_nexthop_lookup_mrib(struct zserv *client,
 	return zebra_server_send_message(client);
 }
 
+int zsend_route_notify_owner(u_char proto, vrf_id_t vrf_id,
+			     struct prefix *p,
+			     enum zapi_route_notify_owner note)
+{
+	struct zserv *client;
+	struct stream *s;
+	uint8_t blen;
+
+	client = zebra_find_client(proto);
```

Comment:
I think `zebra_find_client()` has a problem: it doesn't take the client instance into account. This could easily lead to problems like an OSPF instance receiving notifications from another one.